lib/SPIRV/OCLUtil.h

```diff
@@ -66,6 +66,17 @@ enum OCLMemFenceKind {
   OCLMF_Image = 4,
 };
 
+// This enum declares extra constants for OpenCL mem_fence flag. It includes
+// combinations of local/global/image flags, except global-image combination.
```

Comment:
I'm not sure we should exclude this combination.

Reply:
Indeed, fixed